locales/pl/faq.properties

```diff
@@ -0,0 +1,44 @@
+# faq page
+faq_header=Najczęściej zadawane pytania
+faq_donate_link=wspomóż teraz
+faq_intro=Poniżej znajdują się najczęściej zadawane pytania o przekazywanie datków Fundacji Mozilla. Jeśli nie ma tu odpowiedzi na jakieś pytanie, prosimy napisać na adres <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+faq_item_1_header=W jakie sposoby mogę przekazać datek?
+faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Pod tym adresem można wykonać bezpieczny datek przez Internet za pomocą karty kredytowej/debetowej lub serwisu PayPal</a>. Inne sposoby wsparcia, takie jak czeki, bitcoiny i przelewy bankowe, są opisane na stronie <a href="https://donate.mozilla.org/ways-to-give">Sposoby wsparcia</a>.
+faq_item_2_header=W jaki sposób Mozilla zapewnia bezpieczeństwo informacji o mnie?
+faq_item_2_paragraph_a=Szanujemy prywatność naszych darczyńców i poważnie traktujemy nasze <a href="https://www.mozilla.org/privacy/principles/">zasady prywatności</a>. Mozilla wierzy, że każda osoba powinna mieć kontrolę nad swoimi danymi i utrzymuje wielowarstwowe zabezpieczenia. Więcej informacji znajduje się w <a href="https://www.mozilla.org/privacy/">Polityce prywatności Mozilli</a>.
+faq_item_2_paragraph_b=W pewnych przypadkach, kiedy wpłacasz datek przez firmę trzecią, taką jak <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">PayPal</a> lub <a href="https://stripe.com/privacy/">Stripe</a>, te organizacje także zbierają dane o użytkownikach.
+faq_item_3_header=Po co Mozilli mój adres, aby otrzymać datek?
+faq_item_3_paragraph=Rozumiemy, że Twoja prywatność jest bardzo ważna. Prosimy o minimalny zakres danych wymagany do przetworzenia datków przez karty kredytowe/debetowe, w tym adresy rozliczeniowe. Umożliwia to firmie przetwarzającej płatności zweryfikowanie Twojej tożsamości, przetworzenie płatności i zapobiegnięcie nielegalnych obciążeń karty kredytowej/debetowej. Chronimy prywatność Twoich danych — w razie pytań polecamy przeczytanie naszej <a href="https://www.mozilla.org/privacy/">polityki prywatności</a>. Jeśli nie chcesz wpisywać swoich danych w formularzu, możesz <a href="https://donate.mozilla.org/ways-to-give">wysłać czek</a> lub wesprzeć nas <a href="https://donate.mozilla.org/paypal-donate">przez PayPal</a>.
```

Comment:
"Prosimy o minimalny zakres danych wymagany do przetworzenia datków **dokonowanych po**przez karty kredytowe/debetowe"

"Umożliwia to firmie przetwarzającej płatności zweryfikowanie Twojej tożsamości, przetworzenie płatności i zapobieg**anie nieuprawnionym obciążeniom** karty kredytowej/debetowej"